<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLoadWindow</class>
 <widget class="QMainWindow" name="NewLoadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/yumao.jpg);
QPushButton{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 

QPushButton:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgba(255, 255, 255, 0);
border:2px solid white;border-radius:10px;
}
 
QPushButton:hover{ 
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
}
 

QPushButton:pressed{
background-color: rgba(255, 255, 255, 150);
border:2px solid white;border-radius:10px;
padding-left:6px;
padding-top:6px;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件名:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>普通下载</string>
    </property>
    <property name="icon">
     <iconset resource="pictures.qrc">
      <normaloff>:/new/prefix1/098ad8b4dc34862a4290ddd5e93eb249.png</normaloff>:/new/prefix1/098ad8b4dc34862a4290ddd5e93eb249.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
    <property name="icon">
     <iconset resource="pictures.qrc">
      <normaloff>:/new/prefix1/83f245157afef398ed1830a40b671ad3.jpg</normaloff>:/new/prefix1/83f245157afef398ed1830a40b671ad3.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
    <property name="icon">
     <iconset resource="pictures.qrc">
      <normaloff>:/new/prefix1/300px-131Lapras.png</normaloff>:/new/prefix1/300px-131Lapras.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>391</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>235</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下载到:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>网址:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>多线程下载</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>331</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
